app/helpers/scaptimony_policies_helper.rb

```diff
@@ -4,4 +4,22 @@ def profiles_selection
     return @policy.scap_content.scap_content_profiles unless @policy.scap_content.blank?
     return []
   end
+
+  def submit_or_cancel_policy(f, overwrite = nil, args = { })
+    args[:cancel_path] ||= send("#{controller_name}_path")
+    content_tag(:div, :class => "clearfix") do
+      content_tag(:div, :class => "form-actions") do
+        text    = overwrite ? overwrite : _("Submit")
```

Comment:
It is relevant: It changes the text to "Next". See https://github.com/OpenSCAP/foreman_openscap/pull/35/files#diff-4951ee738829dd0659c417337012b434R9